nixos/tests/installer.nix

```diff
@@ -30,7 +31,10 @@ let
         # To ensure that we can rebuild the grub configuration on the nixos-rebuild
         system.extraDependencies = with pkgs; [ stdenvNoCC ];
 
-        ${optionalString systemdStage1 "boot.initrd.systemd.enable = true;"}
+        ${optionalString systemdStage1 "boot.initrd.systemd = {
+          enable = true;
+          ${optionalString clevisTest "network.enable = true;"}
```

Comment:
Why use `boot.initrd.systemd.network.enable` and not `boot.initrd.network.enable`? It should work for both systemd initrd and scripted initrd to just use `boot.initrd.network.enable`. The `boot.initrd.systemd.network.enable` option is for people who want `networkd` but don't want the default configuration things that nixos would set up. You've already got the other one below for scripted initrd; might as well remove this and make that unconditional.